### DISCUSSION TITLE
Welcome to coreminer Discussions!

### DISCUSSION THREAD

**Discussion by rastislavcore** [catchthatrabbit/coreminer#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hublon-CTNXCB** [comment]
Hello I am very happy to hear from you. I am happy to see that the amount of mining is increasing day by day. I have been exchanging information with people who have 10 mining machines in the Japanese core open chat, which has about 800 members.
So everyone there is digging about 600 pieces and I am a little less with 520 pieces. I checked in the store and everything is still active.

I will send you a picture so you can take a look at it, I know you are busy. I hope everything is ok....thank you in advance.

**Reply by Adariles-Cardi** [reply]
can we have the same worker running off different machines, ? OS? say one in Linux, Mac OS, and Win 11?

**Comment by Hublon-CTNXCB** [comment]
I do not know. I don't think I can tamper with the worker.

iPhoneから送信

> 2022/06/10 1:41、Carlos Tenorio Amariles ***@***.***>のメール:
> 
> can we have the same worker running off different machines, ? OS? say one in Linux, Mac OS, and Win 11?

**Comment by mrmike15** [comment]
In linux ubuntu how do you add a auto start to it..

**Reply by rastislavcore** [reply]
Explained here: https://github.com/catchthatrabbit/coreminer/blob/master/docs/AUTOSTART-LINUX.md
